client/src/Profile.js

```diff
@@ -0,0 +1,118 @@
+import ReactDOM from 'react-dom';
+import React from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import TextField from '@material-ui/core/TextField';
+import Grid from '@material-ui/core/Grid';
+import IconButton from '@material-ui/core/IconButton';
+import CreateIcon from '@material-ui/icons/Create';
+import CheckIcon from '@material-ui/icons/Check';
+import Button from '@material-ui/core/Button';
+import './Profile.css';
+
+const state = {
+    isDisable: true
+}
+
+const styles = makeStyles(theme => ({
+    root: {
+        justifyContent: 'center',
+    },
+    paper: {
+        margin: theme.spacing(15, 8),
+        isplay: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+    },
+    button: {
+        margin: theme.spacing(3, 0, 2),
+    },
+    form: {
+        width: '100%', // Fix IE 11 issue.
+        marginTop: theme.spacing(1),
+    },
+}));
+
+const handleEdit = () => {
```

Comment:
The `Edit` icon looks good. Is it possible to change the text from gray to black when clicking the `Edit` button?

Reply:
yes sir